x-pack/plugins/apm/public/components/shared/Links/ViewMLJob.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiLink } from '@elastic/eui';
+import { Location } from 'history';
+import React from 'react';
+import { getKibanaHref } from './url_helpers';
+
+interface Props {
+  serviceName: string;
+  transactionType?: string;
+  location: Location;
+}
+
+export const ViewMLJob: React.SFC<Props> = ({
```

Comment:
`ViewMLJob` sounds like a method name (`${verb}${object}`). Should name components something like `MLJobLink` to reinforce that it's a component.

Reply:
True, also aligns better with `DiscoverLink` and `KibanaLink`. Renaming to `MLJobLink`